src/strict/api/routes.py

```
@@ -115,3 +125,38 @@ async def process_request(
+                processor = await processor_manager.get_processor(request)
+
+                # Stream results back
+
+                async for chunk in processor.stream_process(request):
+                    await websocket.send_json({"type": "chunk", "content": chunk})
+
+                await websocket.send_json({"type": "done", "content": ""})
+
+            except ValidationError as e:
+                await websocket.send_json({"type": "error", "content": str(e)})
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)